api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -19,6 +19,7 @@
 import java.util.stream.StreamSupport;
 import javax.mail.internet.AddressException;
 import javax.mail.internet.InternetAddress;
+import liquibase.util.BooleanUtils;
```

Comment:
Importing this utility from liquibase seems odd - did you mean a different one? is this supposed to be for null safe handling?

Another way without any utilities: `Boolean.TRUE.equals(...)`

Reply:
ah, wrong package. Should be `import org.apache.commons.lang3.BooleanUtils;`